### PR TITLE
Fix broken pull requests links

### PR DESCRIPTION
**Pull request by rbgnk**
### Why:

Some links for the Pull Requests documentation are missing the closing parenthesis.

### What's being changed:

The broken links had the closing parenthesis added so that they render correctly.

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Review by ramyaparimi — APPROVED** [review]
@rbgnk 
Thanks so much for opening a PR with this fix! I'll update the branch and get this merged once tests are passing 🍏

**Review by meshl555 — COMMENTED** [review]
None .

File: content/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/merging-a-pull-request.md [hunk 1]
```diff
@@ -103,4 +103,4 @@ gh pr merge 523 --squash --body "my squash commit" --delete-branch
 - "[Reverting a pull request](/articles/reverting-a-pull-request)"
 - "[Syncing your branch](/desktop/guides/contributing-to-projects/syncing-your-branch/)" using {% data variables.product.prodname_desktop %}
 - "[About pull request merges](/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/about-pull-request-merges)"
-- "[Addressing merge conflicts](/github/collaborating-with-pull-requests/addressing-merge-conflicts"
+- "[Addressing merge conflicts](/github/collaborating-with-pull-requests/addressing-merge-conflicts)"
```
Comment (meshl555) [inline content/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/merging-a-pull-request.md:106]: None .

**Review by meshl555 — COMMENTED** [review]
None .

**Pull request merged**